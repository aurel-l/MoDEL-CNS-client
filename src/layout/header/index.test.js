import React from 'react';
import { shallow } from 'enzyme';

import Header from '.';

describe('<Header />', () => {
  it('should render', () => {
    expect(shallow(<Header />)).toMatchSnapshot();
  });
});
