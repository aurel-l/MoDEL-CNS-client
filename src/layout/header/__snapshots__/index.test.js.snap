// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render 1`] = `
<WithStyles(ForwardRef(AppBar))
  position="sticky"
>
  <WithStyles(ForwardRef(Toolbar))
    className="toolbar"
  >
    <Link
      title="home"
      to="/"
    >
      <picture>
        <source
          srcSet="logo.webp"
          type="image/webp"
        />
        <LazyImg
          alt="Logo - Link to home page"
          height="103px"
          loading="lazy"
          src="logo.png"
          width="200px"
        />
      </picture>
    </Link>
    <_default />
  </WithStyles(ForwardRef(Toolbar))>
</WithStyles(ForwardRef(AppBar))>
`;
